docs/CONFIG-VARS.md

```diff
@@ -387,3 +387,101 @@ postgres_servers = {
   }
 }
 ```
+
+## Azure Application Gateway with Azure Web Application Firewall (WAF)
```

Comment:
You could add an entry in the table contents to point to this header

Reply:
Added the header link.